packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify.stories.js

```diff
@@ -200,6 +200,19 @@ class DataHydrator extends Component {
                   </button>
                 </td>
               </tr>
+              <tr>
+                <td>Track names, multiple but same as album artist (should be omitted)</td>
+                <td>
+                  <button
+                    type="button"
+                    className="btn btn-lg btn-primary"
+                    data-identifier="lp_emperor-concerto_ludwig-van-beethoven-arthur-rubinstein-bos"
+                    onClick={this.getItem}
+                  >
+                  lp_emperor-concerto_ludwig-van-beethoven-arthur-rubinstein-bos
```

Comment:
Is this a demo file?

Reply:
yep, for the stories. i want to continue to capture live examples and cases to test against